build.gradle

```diff
@@ -23,6 +23,7 @@ ext.bndWorkspace = new Workspace(rootDir, bnd_cnf)
 if (bndWorkspace == null) {
   throw new GradleException("Unable to load workspace ${rootDir}/${bnd_cnf}")
 }
+bndWorkspace.setOffline(project.gradle.startParameter.offline)
```

Comment:
The non-workspace model doesn't have 1) a workspace 2) repos since deps are
managed in gradle, offline already works.

On Aug 31, 2016 12:03 PM, "BJ Hargrave" <notifications@github.com> wrote:

> In build.gradle
> <https://github.com/bndtools/bnd/pull/1665#discussion_r77016063>:
>
> > @@ -23,6 +23,7 @@ ext.bndWorkspace = new Workspace(rootDir, bnd_cnf)
> >  if (bndWorkspace == null) {
> >    throw new GradleException("Unable to load workspace ${rootDir}/${bnd_cnf}")
> >  }
> > +bndWorkspace.setOffline(project.gradle.startParameter.offline)
>
> Why are you doing it here? I would think this change needs to be isolated
> to the bnd plugin rather than requiring everyone edit their root project
> build.gradle file.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bndtools/bnd/pull/1665/files/54b2d4d44c199c68f9fe40c5cd735018200c2cba#r77016063>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAI9TAdWxDAQ5dlJ-weqrrDP9a96miymks5qlaTHgaJpZM4JxuYl>
> .
>